changelog.d/13867.misc

```diff
@@ -0,0 +1 @@
+Correct the comments in the complement dockerfile.
```

Comment:
The previous changelog and PR description could use some updating as well, see my comments over there, https://github.com/matrix-org/synapse/pull/13819#discussion_r978065151